packages/application-config/custom-view.schema.json

```diff
@@ -0,0 +1,245 @@
+  "properties": {
+    "name": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "string"
+    },
+    "description": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "string"
+    },
+    "cloudIdentifier": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "string"
+    },
+    "mcApiUrl": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "string"
+    },
+    "oAuthScopes": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "object",
+      "properties": {
+        "view": {
+          "description": "See https://docs.commercetools.com/TODO",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string",
+            "pattern": "view_(.*)"
+          },
+          "uniqueItems": true
+        },
+        "manage": {
+          "description": "See https://docs.commercetools.com/TODO",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string",
+            "pattern": "manage_(.*)"
+          },
+          "uniqueItems": true
+        }
+      },
+      "additionalProperties": false,
+      "required": ["view", "manage"]
+    },
+    "additionalOAuthScopes": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "array",
+      "default": [],
+      "uniqueItems": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "See https://docs.commercetools.com/TODO",
+            "type": "string"
+          },
+          "view": {
+            "description": "See https://docs.commercetools.com/TODO",
+            "type": "array",
+            "default": [],
+            "items": {
+              "type": "string",
+              "pattern": "view_(.*)"
+            },
+            "uniqueItems": true
+          },
+          "manage": {
+            "description": "See https://docs.commercetools.com/TODO",
+            "type": "array",
+            "default": [],
+            "items": {
+              "type": "string",
+              "pattern": "manage_(.*)"
+            },
+            "uniqueItems": true
+          }
+        },
+        "additionalProperties": false,
+        "required": ["name", "view", "manage"]
+      }
+    },
+    "env": {
+      "description": "See https://docs.commercetools.com/TODO",
+      "type": "object",
+      "properties": {
+        "development": {
+          "type": "object",
+          "properties": {
+            "initialProjectKey": {
+              "description": "See https://docs.commercetools.com/TODO",
+              "type": "string"
+            },
+            "teamId": {
+              "type": "string"
+            },
+            "hostUriPath": {
+              "description": "See https://docs.commercetools.com/TODO",
+              "type": "string"
+            }
```

Comment:
I've been thinking about this (originally it was `hostUrl`) and opted to change it to `hostUriPath` as the URL origin is actually irrelevant and users don't have to worry about that.

I also don't expect users to use this much, as we now also provide a computed value for development

```ts
const defaultHostUriPath = oidcConfig.initialProjectKey
  ? `/${oidcConfig.initialProjectKey}/${entryPointUriPath}`
  : `/${entryPointUriPath}`;
```